### PR TITLE
Separated the exercise versions array into its own API

### PR DESCRIPTION
**Pull request by Dantemss**

### CODE REVIEW

File: app/representers/api/v1/exercises/representer.rb [hunk 1]
```diff
@@ -49,14 +49,6 @@ class Representer < BaseRepresenter
                  required: true
                }
 
-    collection :versions,
-             type: Array,
-             writeable: false,
-             readable: true,
-             getter: ->(user_options:, **) do
-               visible_versions(can_view_solutions: SOLUTIONS.call(user_options: user_options))
-             end
```
Comment (jpslav) [inline app/representers/api/v1/exercises/representer.rb]: Leave it in so as not to make a breaking API change?  If take out, need to double check that CNX isn't using this.

Comment (Dantemss) [reply app/representers/api/v1/exercises/representer.rb]: I will check with Ross. But this has to wait for FE anyway.

Comment (Dantemss) [reply app/representers/api/v1/exercises/representer.rb]: Ross says @philschatz wrote the CNX code. But the thinks it's not used.

**Comment by Dantemss** [comment]
This would fix a bug when navigating exercises version using the arrow buttons.

**Comment by Dantemss** [comment]
Closing this one because I need another uncached field (delegates) so I'll just make versions and delegates not cached, but keep them in the same API.

**Pull request closed without merging**